storage/cache/tree_layout.go

```diff
@@ -0,0 +1,117 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cache
+
+import (
+	"fmt"
+
+	"github.com/google/trillian/storage"
+)
+
+const (
+	// depthQuantum defines the smallest supported subtree height, which all
+	// subtree heights must also be a multiple of.
+	//
+	// WARNING: The code below breaks if this value is not a multiple of 8,
+	// because it uses storage.NodeID byte representation directly.
+	depthQuantum = 8
```

Comment:
It's an old comment but I don't think changing that value is safe at all. It might work for maps but I suspect it will cause problems for logs if you set it to e.g 16.

Reply:
Yeah, I don't think it's a good idea to ever change it. The comment only focuses on `TreeLayout` though, which in isolation doesn't break if this constant changes to another multiple of 8. I updated the comment.